cfgov/ask_cfpb/jinja2/ask-cfpb/landing-page.html

```diff
@@ -49,50 +49,789 @@ <h2 class="u-show-on-mobile u-mt45">
 
             <div class="u-hide-on-mobile question-categories u-mb15">
 
+                
                 {#
-                    TODO: This can use the standard card-group.html template
-                          if the partially styled links are removed.
+                    TODO:   This can use the standard card-group.html template
+                            if the partially styled links are removed.
                 #}
+
                 <div class="o-card-group o-card-group__column-2">
                     <div class="o-card-group_cards">
 
-                    {% for portal in portal_cards %}
+                        {% if page.language == 'en'%}
+                        <article class="m-card">
+                            <h2 class="m-card_heading">
+                                <a class="" href="/consumer-tools/auto-loans/">
+                                    {{ svg_icon( "car" ) }}
+                                    <span>Auto loans</span>
+                                </a>
+                            </h2>
+
+                            <ul class="m-list">
+                                
+                                    <li class="m-list_item">
+                                    <a class="a_link a-link__partially-styled" href="/ask-cfpb/what-effect-will-shopping-for-an-auto-loan-have-on-my-credit-en-763/">
+<span class="a-link__partially-styled_plain">What effect will shopping for an auto loan have on my credit? </span>
+<span class="a-link__partially-styled_underlined">Read answer</span>
+</a>
```

Comment:
I note your comment about this hardcoding being only temporarily; if so, what do you think about implementing this in Python instead via a hardcoded list of Answer IDs. See implementation [here](https://github.com/cfpb/consumerfinance.gov/commit/c13a3d54b0ddc005c739a8b4276546215f4f71a0). This lets you leave the landing-page.html as-is before your PR, without duplicating the various boilerplate for each link.

Reply:
Love it -- merging these changes!